### PR TITLE
introduce brands section

### PR DESCRIPTION
**Pull request by mikkelklokkerud**
Create brands section as per figma design and source data from new sanity fields.

### CODE REVIEW

**Comment by klyngen** [comment]
💻 https://brandsw-846.review.test-alv.no

**Comment by klyngen** [comment]
💻 https://brandsw-846.review.test-alv.no

File: packages/shared-components/src/index.js [hunk 1]
```diff
@@ -1,39 +1,38 @@
-export * as Form from './components/form';
```
Comment (TheKnarf) [inline packages/shared-components/src/index.js:1]: A few to many changes in this file perhaps that are unrelated to the PR?

Comment (mikkelklokkerud) [reply packages/shared-components/src/index.js:1]: Yes, my linter decided to throw a party

Comment (mikkelklokkerud) [reply packages/shared-components/src/index.js]: It decided to organise imports alphabetically - this change is now undone

**Comment by klyngen** [comment]
💻 https://brandsw-846.review.test-alv.no

File: packages/website/src/pages/om-oss.js [hunk 2]
```diff
@@ -68,6 +75,25 @@ async function getAboutContent() {
         aboutPage: allAboutPage {
           pageDescription
           pageTitle
+          brands {
```
Comment (klyngen) [inline packages/website/src/pages/om-oss.js:78]: The query for brands should be extracted to a constant. That will make it easier to maintain in the future.  Basically like this:

```typescript
const brandQuery = `brands {
            headingRaw
            buttonText
            buttonLink
            logos {
              alt
              image {
                asset {
                  id: _id
                  metadata {
                    dimensions {
                      height
                      width
                    }
                  }
                }
              }
            }
          }`;

```

const someQuery = gql`
  ...otherGqlStuff
  ${brandQuery}
`

File: packages/cms/schemas/brands.js [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+      of: [{ type: 'imageWithAlt' }],
+      validation: (Rule) => Rule.required().min(1),
+    },
+    {
+      title: 'Button text',
+      name: 'buttonText',
+      type: 'string',
+    },
+    {
+      title: 'Button link',
```
Comment (klyngen) [inline packages/cms/schemas/brands.js:24]: Does it make sense to have a link without a URL? Should this be required?

File: packages/website/src/pages/om-oss.js (hunk 2 above)
Comment (mikkelklokkerud) [reply packages/website/src/pages/om-oss.js:78]: Is this a new pattern we want to adopt? Or are we doing this already somewhere in our codebase - if so it would be useful to look at where/how we store and possibly import the query contants. Can think of a few various ways

Comment (mikkelklokkerud) [reply packages/website/src/pages/om-oss.js:78]: Regardless, it's a good idea.. I want a codemod to apply this everywhere in the codebase....

Comment (klyngen) [reply packages/website/src/pages/om-oss.js]: No, don't think it is a pattern. But put them somewhere central, maybe even close to the hookQueries. You can call them queryFragments or something. That will greatly improve the codebase :+1:

**Comment by klyngen** [comment]
💻 https://brandsw-846.review.test-alv.no

**Review by klyngen — APPROVED** [review]

**Comment by klyngen** [comment]
💻 https://brandsw-846.review.test-alv.no
